_about/release-cadence.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Build & Release Cadence
+description: How we manage, number, and support Istio releases.
+weight: 5
+---
+
+We produce new builds of Istio on a daily basis. Around once a month or so, we take one of these daily
+builds and run it through a number of additional qualification tests and tag the build as an Alpha release.
+Around once a quarter or so, we take one of these Alpha releases, run through a bunch more tests
+and tag the build as a Long Term Support (LTS) release. Finally, if we find something wrong with an Alpha
+or LTS release, we issue patches.
```

Comment:
historically we have not patched the monthly release after the first qualified one. not sure if we want to commit to fixing critical issues in monthlies vs ask people to wait for the next one

Reply:
I'd almost say we don't bother mentioning patching of the monthlies. Over time we don't want to spend time doing that, those patches should just roll into a new monthly RC.